app/views/components/_document-list.html.erb

```diff
@@ -0,0 +1,50 @@
+<%
+  group_index ||= false
+  track_clicks = "data-module=track-click" if group_index
+  dimension28 = items.count.to_s
+%>
+<ol class="app-c-document-list" <%= track_clicks %>>
+  <% items.each_with_index do |link, link_index| %>
+    <li class="app-c-document-list__item">
+      <h3 class="app-c-document-list__item-title">
```

Comment:
I've seen something similar (which I think the link above is implying) but a bit less restrictive - more that if an element inside a block can be moved around freely without relying on any parent CSS, then it doesn't need to be nested.